modules/social_features/social_profile/modules/social_profile_preview/social_profile_preview.info.yml

```diff
@@ -0,0 +1,7 @@
+name: 'Social Profile Preview'
+description: 'Provides a modal window to display profile details.'
+type: module
```

Comment:
I'd say that experimental modules shouldn't be hidden because in this case, they will not be visible on the extensions overview page and clients can not try how these modules work.

Reply:
I agree with @OH17L. It does not make sense to release this experimental module if clients won't able to enable it.